support-frontend/assets/helpers/stripe.js

```diff
@@ -23,6 +23,8 @@ export const stripeCardFormIsIncomplete = (
   contributionType: ContributionType,
   paymentMethod: PaymentMethod,
   stripeCardFormComplete: boolean,
-): boolean => contributionType === 'ONE_OFF' &&
-    paymentMethod === Stripe &&
-    !(stripeCardFormComplete);
+  stripeElementsRecurringTestVariant: string,
+): boolean =>
+  (contributionType === 'ONE_OFF' || stripeElementsRecurringTestVariant === 'stripeElements') &&
```

Comment:
Would be good to have an enum for contribution types to avoid using literal values and comparisons: https://flow.org/en/docs/types/utilities/

Reply:
Ideally I'd be able to use something like `contributionType === ContributionType.ONE_OFF` here, but it doesn't work despite having a [ContributionType](https://github.com/guardian/support-frontend/blob/master/support-frontend/assets/helpers/contributions.js#L28) type. I could also add an enum with the same values, though it's a bit rubbish

Reply:
actually I'm not sure there's anything more we can do here with flow, unless I'm missing something